src/components/InputFilters/InputFilters.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import Form from 'react-bootstrap/Form';
+import '../../styles/input-filters.scss';
+
+function InputFilters({ isOpen, setFilteredText }) {
+  const inputRef = useRef();
+
+  useEffect(() => {
+    if (isOpen) inputRef.current?.focus();
```

Comment:
We should always use brackets

```suggestion
    if (isOpen) {
      inputRef.current?.focus();
    }
```